src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -1260,8 +1261,11 @@ public Invite createInvite(JSONObject object)
         final long guildId = guildObject.getLong("id");
         final String guildName = guildObject.getString("name");
         final String guildSplashId = guildObject.optString("splash", null);
+        final VerificationLevel guildVerificationLevel = VerificationLevel.fromKey(guildObject.optInt("verification_level", -1));
+        final int presenceCount = object.optInt("approximate_presence_count", -1);
+        final int memberCount = object.optInt("approximate_member_count", -1);
```

Comment:
Should this be in the guild object if discord doesn't provide them in the guild object?

Reply:
It's not provided in the guild object, but from a library user's standpoint I think it might make more sense. Either way would be fine from an implementation standpoint though.

Reply:
I'm fine with either, too